test/test_jit.py

```diff
@@ -2581,6 +2581,120 @@ def forward(self, x):
 
 
 class TestScript(JitTestCase):
+
```

Comment:
With so many test case added, I think it is worth to make its own test file, search for `test_enum.py` for an example.